src/lib.rs

```diff
@@ -420,7 +420,7 @@ pub trait ScancodeSet {
 }
 
 /// The set of modifier keys you have on a keyboard.
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq, Hash)]
```

Comment:
If this type is `Copy` then `fn get_modifiers()` should just return a copy and not a reference. But I think because it's eight bytes, it shouldn't be `Copy` and `get_modifiers` should be left as-is.

The best solution is probably to actually hide the flags inside in a bitfield and add getter and setter methods for each of them - the object would then only be 1 byte (maybe 2). But that's a breaking change we can save for another day.

Reply:
Fair point, I will remove the Copy derive!
A bitflag (u16 atleast) would be a nice idea for the future.